haystack/preview/components/audio/whisper_remote.py

```diff
@@ -54,6 +54,12 @@ def __init__(
         self.whisper_params = whisper_params or {}
 
         self.model_name = model_name
+        self.init_parameters = {
+            "api_key": self.api_key,
+            "model_name": self.model_name,
+            "api_base": self.api_base,
+            "whisper_params": self.whisper_params,
+        }
```

Comment:
All other components should need this as well, right? :eyes:

Reply:
How comes that tests are passing? I thought we were testing serialization for all of them :thinking:

Reply:
This is the only component that doesn't have a default in `__init__`, other all have defaults.
The other components that can be converted to dict and back but we lose information, I'll update them in later PRs.

Also this means that testing serialisation like that doesn't work.

Reply:
Oh! Right I see. Let's open an issue about these tests, we should fix them

Reply:
Already created #5550 and deepset-ai/canals#81. 👍